command/push_test.go

```diff
@@ -323,12 +322,11 @@ func TestPush_preferAtlas(t *testing.T) {
 		t.Fatalf("bad: %#v", client.UpsertOptions)
 	}
 
-	variables := map[string]interface{}{
-		"foo": "old",
-		"bar": "foo",
-	}
+	variables := pushTFVars()
+	variables["foo"] = "old"
+
 	if !reflect.DeepEqual(client.UpsertOptions.Variables, variables) {
-		t.Fatalf("bad: %#v", client.UpsertOptions)
+		t.Fatalf("bad: %#v", client.UpsertOptions.Variables)
```

Comment:
Nope, this is another test, which we'll fix later. I only added `.Variables` for a shorter error message.